### PR TITLE
feature/49-2-refactoring-create-doors-and-levels-game

### PR DESCRIPTION
**Pull request by vsatkov**
issue #49 part 2

### CODE REVIEW

File: HW2/Valeriy_Satkov/CreateDoorsAndLevels/CreateDoorsAndLevels/GameSettings.json [hunk 1]
```diff
@@ -2,5 +2,6 @@
   "DoorsAmount": 5,
```
Comment (bartimys1) [inline HW2/Valeriy_Satkov/CreateDoorsAndLevels/CreateDoorsAndLevels/GameSettings.json]: DoorsAmount can be more than 10

`for (int i = 0; i < doorsAmount - 1; i++)
            {
                while (result.Contains(randomNumber = random.Next(1, 9))) ;
                result.Add(randomNumber);         
 }`
you have an endless cycle

Comment (MilRoad) [reply HW2/Valeriy_Satkov/CreateDoorsAndLevels/CreateDoorsAndLevels/GameSettings.json]: I agree. I think it's okay if doors' numbers repeat. 
But DoorsAmount can be whatever we want.

**Comment by MilRoad** [comment]
I think that everything is fine and language problem is solved.
But I didn't notice that your numbers generator makes non-repeating doors and doesn't work if doors amount more than 10.
Fix it and that's all :)

File: HW2/Valeriy_Satkov/CreateDoorsAndLevels/CreateDoorsAndLevels/GameSettings.json (hunk 1 above)
Comment (vsatkov) [reply HW2/Valeriy_Satkov/CreateDoorsAndLevels/CreateDoorsAndLevels/GameSettings.json]: It is a big omission on my part. Thanks

File: HW2/Valeriy_Satkov/CreateDoorsAndLevels/CreateDoorsAndLevels/Modules/DoorsNumbersGenerator.cs [hunk 2]
```diff
@@ -14,7 +14,7 @@ public List<int> generateDoorsNumbers(int doorsAmount)
 
             for (int i = 0; i < doorsAmount - 1; i++)
             {
-                while (result.Contains(randomNumber = random.Next(1, 9))) ;
+                while (result.Contains(randomNumber = random.Next(1, doorsAmount * 2)));
```
Comment (MilRoad) [inline HW2/Valeriy_Satkov/CreateDoorsAndLevels/CreateDoorsAndLevels/Modules/DoorsNumbersGenerator.cs]: Now we can create doors with numbers more than 9 :(

Comment (vsatkov) [reply HW2/Valeriy_Satkov/CreateDoorsAndLevels/CreateDoorsAndLevels/Modules/DoorsNumbersGenerator.cs]: Yeah, but it's not a problem. I didn't find any requirements about it.

**Pull request merged**